exhibitor-core/src/main/java/com/netflix/exhibitor/core/azure/AzureClient.java

```diff
@@ -0,0 +1,23 @@
+package com.netflix.exhibitor.core.azure;
+
+import com.microsoft.azure.storage.blob.BlobProperties;
+import com.microsoft.azure.storage.blob.CloudBlob;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.ListBlobItem;
+
+import java.io.Closeable;
+
+public interface AzureClient extends Closeable {
+    public CloudBlobClient getClient() throws Exception;
+
+    public CloudBlob getBlob(String containerName, String uri) throws Exception;
```

Comment:
if this is a new interface (I assume, as it doesn't extend anything?) please use the URI or URL classes instead of a String